src/exercises/exerciseTypes/NextNavigation.js

```diff
@@ -252,6 +253,7 @@ export default function NextNavigation({
                 reload={reload}
                 setReload={setReload}
                 notifyDelete={() => setIsDeleted(true)}
+                notifyWordChange={() => isBookmarkChanged()}
```

Comment:
Ehi Tiago, could this be a sign that we should move the edit button away from the next navigation? Or rename NextNavigation to something else? Currently it feels wrong to pass the `notifyWordChange` to the `NextNavigation` which passes it on...

I think the right direction would be to move both the speech button and the edit button. We've already talked about the speech into the main exercise as opposed to having them in the `NextNavigation`:

<img width="631" alt="image" src="https://github.com/user-attachments/assets/019128bb-cd08-4684-8159-934fb07c6a29">

We can almost not pass the `exerciseBookmark` down to the `NextNavigation` too... 

In any case, I would say, the first step could be moving the Edit bookmark button, so the button lives in the same component as the bookmarks that it edits. 

Do you see what I mean? 

P.S. Surely, we could also, just as well not do it now, but rather open an issue instead.

Reply:
Let's merge this for now because it's a bug fix. Then we'll do the refactoring at a later point. I've created an issue with this conversation: https://github.com/zeeguu/web/issues/535.